support-frontend/assets/helpers/productCatalogToProductOption.ts

```diff
@@ -141,22 +109,7 @@ export const getProductOptionFromProductAndRatePlan = (
 			return getPaperProductOptions(ratePlanKey);
 	}
 };
-
-export {
-	NoProductOptions,
-	Saturday,
-	SaturdayPlus,
-	Sunday,
-	SundayPlus,
-	Weekend,
-	WeekendPlus,
-	Sixday,
-	SixdayPlus,
-	Everyday,
-	EverydayPlus,
-	NewspaperArchive,
-	ActivePaperProductTypes,
-	paperProductsWithDigital,
-	paperProductsWithoutDigital,
-	productOptionIfDigiAddOnChanged,
-};
+export { productOptionIfDigiAddOnChanged };
+export { paperProductsWithoutDigital };
+export { paperProductsWithDigital };
+export { ActivePaperProductTypes };
```

Comment:
Very minor but is there a reason not to export these all together?

```suggestion
export {
  productOptionIfDigiAddOnChanged,
  paperProductsWithoutDigital,
  paperProductsWithDigital,
  ActivePaperProductTypes,
};
```